packages/slice-machine/components/ChangesItems/ChangesItems.tsx

```diff
@@ -116,30 +115,22 @@ export const ChangesItems: React.FC<ChangesItemsProps> = ({
             gridGap="32px 16px"
             elems={unSyncedSlices}
             gridTemplateMinPx="290px"
-            defineElementKey={(slice: ComponentUI) => slice.model.name}
-            renderElem={(slice: ComponentUI) => {
-              return SharedSlice.render({
-                showActions: true,
-                slice: slice,
-                wrapperType:
-                  modelsStatuses.slices[slice.model.id] !== ModelStatus.Deleted
-                    ? WrapperType.clickable
-                    : WrapperType.nonClickable,
-                StatusOrCustom: {
-                  status: modelsStatuses.slices[slice.model.id],
-                  authStatus,
-                  isOnline,
-                },
-                actions: {
-                  onUpdateScreenshot: (e: React.MouseEvent) => {
-                    e.preventDefault();
+            defineElementKey={(slice) => slice.model.name}
+            renderElem={(slice) => {
+              const modelStatus = modelsStatuses.slices[slice.model.id];
+              return (
+                <SharedSliceViewCard
```

Comment:
❓ It's what you want to have 3 slices in the row?
Asking since it's the only place it's not 2

<img width="993" alt="Screenshot 2023-10-05 at 10 47 29" src="https://github.com/prismicio/slice-machine/assets/19946868/3b6e9f17-8ef4-4fca-acc6-e934e5e9cc22">